crates/amaru/tests/evaluate_ledger_states.rs

```diff
@@ -0,0 +1,223 @@
+// Copyright 2024 PRAGMA
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use amaru_kernel::{
+    AuxiliaryData, Epoch, EraHistory, Hasher, KeepRaw, MintedTransactionBody, MintedTx,
+    MintedWitnessSet, Point, PoolId, TransactionPointer, cbor, network::NetworkName,
+    protocol_parameters::ProtocolParameters,
+};
+use amaru_ledger::{
+    self,
+    context::DefaultValidationContext,
+    rules::transaction,
+    state::{self, volatile_db::StoreUpdate},
+    store::{EpochTransitionProgress, ReadStore, Store, TransactionalContext},
+};
+use amaru_stores::rocksdb::RocksDB;
+use pallas_addresses::Network;
```

Comment:
This is already re-exported from amaru_kernel, you shouldn't introduce pallas_addresses here.